cloud_storage/google_cloud_storage/upload_file.py

```diff
@@ -0,0 +1,255 @@
+import os
+import sys
+import re
+import argparse
+import glob
+
+from gcloud import storage
+from gcloud.exceptions import *
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--bucket-name', dest='bucket_name', required=True)
+    parser.add_argument('--force-bucket-creation', dest='force_create',
+            action='store_true', required=False)
```

Comment:
Per our coding guidelines, arguments should only accept text values. Because of the way things are designed in Shipyard, we're unable to support flags whose presence == True and whose lack of presence == False.

Instead, _if_ we were to include something like this, you would have to pass through a text value that gets re-interpreted into a boolean within the script. i.e. "Yes/No,True/False,etc."

 https://www.notion.so/shipyardapp/Coding-Guidelines-9b39355f98f4476fa5e5f53ec72d609c#fbcfadac324d4c8e92638a92eaea5f2c